model/roots.js

```diff
@@ -51,9 +51,31 @@ async function getVerseRootWords(data) {
         })
     );
 }
+async function getRootsMeaning(data){
+    const sentences = [];
+    var all_roots = getVerseRootWords(data);
+    for (let root of all_roots){
+        let word = root.word                //get the word of the root (idk how to access the variable)
+        let rootmeaning = root.description  //get the meaning of the root  (idk how to access the variable)
+        let sentence  = "The word" + word + "comes from the root" + root + "and is associated with the meanings of:" + rootmeaning
```

Comment:
Javascript allows you to use variable names inside of strings without using the `+` for concatenation, it's a cool and useful feature that makes writing code a little cleaner. I'll change it to that format so maybe you can check it out later if you want.

Reply:
Your for loop is essentially doing the right thing though, just thought you might find the other technique cool/useful after it's implemented.